### ISSUE TITLE
Using Netlify CMS for Static Sites

### ISSUE THREAD

**Issue by XertroV** [voteflux/flux-docs#2]
I've been watching Netlify's CMS for static sites for a while (https://www.netlifycms.org/)

The idea is it integrates with various static site generators (jekyll / hugo / etc) and allows non-git ppl to edit the source for a static site in a user friendly way.

> Netlify CMS is adaptable to a wide variety of projects. The only inflexible requirement is that your site content must be written in markdown, JSON, YAML, or TOML files, stored in a repo on GitHub.

So it _sounds_ like it should work with sphinx...

This issue is to track discussion and possibility of using this to massively simplify workflow for non-techy-ppl

@pwhipp - CCing you on this bc I think it was mentioned you were willing to train people or that sort of thing. I noticed you had instructions on python stuff in the readme, so this sort of thing could help a bunch to keep it KISS for new ppl

**Comment by pwhipp** [comment]
Netifly might be the go to update https://voteflux.org/ and it looks worth watching. I love using react/redux for front end work.

For flux-docs, having the documents in the folder structure makes it a repository of documents that can be compiled into many meta documents (and matching websites). With sphinx we can build a pdf/latex/epub of any document at any level as well as have the searchable website with all the content. I'm not sure how important these features will prove to be over time but it does not look like they are on netifly's roadmap.

I'm happy to help with training whatever route we take. I don't expect casual editors to make use of python although a local IDE with rst previewing and live local updates will be a big productivity booster.

**Comment by XertroV** [comment]
> With sphinx we can build a pdf/latex/epub of any document at any level as well as have the searchable website with all the content. I'm not sure how important these features will prove to be over time but it does not look like they are on netlify's roadmap.

Netlify is a CMS so doesn't _really_ care about sphinx, just about generating the markup. If it generates valid sphinx docs then sphinx should be able to do PDFs etc too.

>  I don't expect casual editors to make use of python although a local IDE with rst previewing and live local updates will be a big productivity booster.

👍

**Comment by pwhipp** [comment]
I'll have to play with netifly. CMS often don't do restructuring well e.g change a doc to a folder containing smaller docs and vice versa.

**Comment by XertroV** [comment]
Yeah, have had problems like that in the past - including one bad one that broke a LOT on the current v2 site

**Comment by pwhipp** [comment]
Maybe I should add a document to the repo - something like /notes/selecting_a_static_website_builder... so we can list requirements and rate the options.

**Comment by XertroV** [comment]
Sounds good!
